.github/workflows/e2e-udx.yml

```
@@ -59,6 +66,7 @@ jobs:
         export OPERATOR_IMG=${{ inputs.operator-image }}
         export VLOGGER_IMG=${{ inputs.vlogger-image }}
         export E2E_TEST_DIRS=tests/e2e-udx
+        export VERTICA_DEPLOYMENT_METHOD_OVERRIDE=${{ inputs.vertica-deployment-method }}
```

See my other comment, but I think with those other fixes this can be rewritten like:
```suggestion
        export VERTICA_DEPLOYMENT_METHOD=${{ inputs.vertica-deployment-method }}
```